force-app/main/default/classes/ContactTriggerUtility.cls

```diff
@@ -0,0 +1,26 @@
+public with sharing class ContactTriggerUtility {
+ 
+
+
+
+    public static void afterUpdate(List<Contact> newTrigger, Map<ID, Contact> oldTrigger){
```

Comment:
tutaj to samo, ta metodka nie powinna sie tak nazywac, tylko powinna wskazywac co robi jej ciało